kube/icasework/icasework-resolver-deployment.yml

```diff
@@ -33,8 +33,8 @@ spec:
     spec:
       containers:
         - name: icasework-resolver
-          # release v4.2.0
-          image: quay.io/ukhomeofficedigital/icasework-resolver:e847453544f41a85454e055b5a17b91bfc4dfd70
```

Comment:
you won't be able to merge this because this is using a different icw image

Reply:
I think this will now work with the old icw-resolver

Reply:
ok so we need a new icasework-resolver image?

Reply:
hopefully not